1-development/list-posts-by-rating/src/main/java/com/Joao_Branco/listpostsbyrating/domain/Post.java

```diff
@@ -0,0 +1,91 @@
+package com.Joao_Branco.listpostsbyrating.domain;
+
+import com.Joao_Branco.listpostsbyrating.Exceptions.InvalidVotesNumberException;
+
+public class Post {
+
+    private int postId;
+    private int upvotes;
+    private int downvotes;
+    private int totalVotes;
+    private String postBody;
+
+    public Post(int postId, int upvotes, int downvotes, String postBody) {
+        this.postId = postId;
+        this.setUpvotes(upvotes);
+        this.setDownvotes(downvotes);
+        this.setTotalVotes(upvotes + downvotes);
+        this.setPostBody(postBody);
+    }
+
+    public Post() {
+
+    }
+
+    public int getTotalVotes() {
+        return totalVotes;
+    }
+
+    public void setTotalVotes(int totalVotes) {
+        if (totalVotes >= 0)

```

Comment:
Is there a way to improve the usage of this duplicate code in this class?

Reply:
I guess the most obvious thing I can think of right now is just to delete this set method as it's not really needed.